src/components/pages/order/Main/Panel/PanelTabs.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { useContext } from "react";
+import PanelContext from "../../../../context/OrderContext";
+import { FiChevronDown, FiChevronUp } from "react-icons/fi";
+import styled from "styled-components/macro";
+import Tab from "../../../../reusable-ui/Tab";
+import { theme } from "./../../../../../theme/index";
+import { configTabs } from "./configTabs";
+
+function PanelTabs() {
+  //// revoir les condition true et false de tabicon et add ** edit en position collapsed avec icon
+  const {
+    tabSelected,
+    setTabSelected,
+    tabIconSelected,
+    setTabIconSelected,
+    isCollasped,
+    setIsCollasped,
+  } = useContext(PanelContext);
+
+  const isTabSelected = (tab) => {
+    if (tab === "icon") {
+      setTabIconSelected(!tabIconSelected);
+      setIsCollasped(!isCollasped);
+    }
+    if (tab !== "icon") {
+      setTabSelected(tab);
+    }
+    if ((tabSelected === "add" || tabSelected === "edit") && tabIconSelected) {
+      setIsCollasped(!isCollasped);
+      setTabIconSelected(!tabIconSelected);
+    }
+  };
+
+  const ModifyClassTabSelected = (tab) =>
```

Comment:
```suggestion
  const modifyClassTabSelected = (tab) =>
```

ce n'est pas un composant donc pas de majuscule

Reply:
Exact